episodes/2-identify-the-problem.Rmd

```diff
@@ -42,62 +35,57 @@ rodents <- read_csv("data/surveys_complete_77_89.csv")
 
 ```
 
+The first step we'll cover is what to do when encountering an error or other undesired output from your code. It is our experience that many seemingly-impossible errors can be fixed on the route to create a reproducible example for an expert helper. With this episode, we hope to teach you the basics about identifying how your error might be occurring, and how to isolate the problem for others to look at.
```

Comment:
This chunk explains that you already know you're shooting for a reprex, and this is what happens along the way. But the rest of this episode is oriented toward focusing on error messages first and then determining whether it is desirable to make a reprex. Those are contradictory framings and I think we should pick one (the second one seems good to me and would require less change--just rephrase this part)